address.md

```diff
@@ -0,0 +1,162 @@
+# Address
+
+An address is an identifier that refers to an actor in the Filecoin state. All actors (miner actors, the storage market actor, account actors) have an address. An address encodes information about the network it belongs to, the type of data it contains, the data itself, and depending on the type, a checksum.
+
+The Address interface is as follows:
+
+```go
+type Addressi interface {
+    New(n byte, t byte, data []byte) Addressi
+    Marshal(a Addressi) []byte
+    Unmarshal(data []byte) Addressi
+    EncodeString(a Addressi) string
+    DecodeString(s string) Addressi
+    Checksum(a Addressi) []byte
+    validateChecksum(data, cksm []byte) bool
+}
+```
+
+An exampple implementation of this interface is shown below:
+
+```go
+type Address struct {
+    // 0: main-net
+    // 1: test-net
+    network byte
```

Comment:
when did we decide to stop differentiating between testnets?

Reply:
I don't think we have decided that, what do you mean?

Reply:
@dignifiedquire I don't really think its useful. If many people disagree with that, we could put it in the user visible encoding, but not include it in the bytes representation.

Reply:
We will have support for Mainnet and Testnet in the initial address spec